CharacterCreation/06_Skills.txt

```diff
@@ -51,48 +51,48 @@ often. Talk to your GM if you would like your character to learn a more specific
 case of one of these skills or a skill not listed below.
 
 == Charisma ==
-Deception:
+Deception-

```

Comment:
"Deception:" will parse as a topic header, while "Deception-" won't parse as anything. What was wrong with the colons?

Reply:
Right! I forgot that I used the colons on purpose.